executor/ddl_test.go

```diff
@@ -183,6 +183,15 @@ func (s *testSuite) TestAddNotNullColumnNoDefault(c *C) {
 	tk.MustExec("set sql_mode=''")
 	tk.MustExec("insert nn (c1) values (3)")
 	tk.MustQuery("select * from nn").Check(testkit.Rows("1 0", "2 0", "3 0"))
+
+	tk.MustExec("alter table nn add column (c3 int default -1, c4 int)")
```

Comment:
Please add more tests about different column types, different column positions, and failed cases. Besides, your can create a new test function.